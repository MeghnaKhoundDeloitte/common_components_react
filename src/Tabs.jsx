import React from 'react';

export class Tabs extends React.Component {
  render() {
      return <ul className="myTab">
            {this.props.items.map(current => <li key={current.key}>{current.label}</li>)}
        </ul>
  }
}
