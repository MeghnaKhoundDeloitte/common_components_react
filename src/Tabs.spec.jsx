import React from 'react';
import { shallow } from 'enzyme';
import { Tabs } from './Tabs';

const tabObj= [
  {"key": "1", "label": "tab1", "actionItems": "2", selected:true},
  {"key": "2", "label": "tab2", "actionItems": "1", selected:false},
  {"key": "3", "label": "tab3", "actionItems": "2", selected:false},
  {"key": "4", "label": "tab4", "actionItems": "4", selected:false}
];

test('Tabs renders correctly', () => {
  const component = shallow(<Tabs items={tabObj}/>);
  expect(component).toMatchSnapshot();
})
