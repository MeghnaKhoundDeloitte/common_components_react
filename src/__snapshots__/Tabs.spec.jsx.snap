// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders correctly 1`] = `
<ul
  className="myTab"
>
  <li>
    tab1
  </li>
  <li>
    tab2
  </li>
  <li>
    tab3
  </li>
  <li>
    tab4
  </li>
</ul>
`;
